src/Jackett.Common/Indexers/ShizaProject.cs

```diff
@@ -0,0 +1,202 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Html.Parser;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace Jackett.Common.Indexers
+{
+    [ExcludeFromCodeCoverage]
+    internal class ShizaProject : BaseWebIndexer
+    {
+        public ShizaProject(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
+            : base(id: "ShizaProject",
+                   name: "ShizaProject",
+                   description: "ShizaProject Tracker is a semi-private russian tracker and release group for anime",
+                   link: "http://shiza-project.com/",
+                   caps: new TorznabCapabilities(),
+                   configService: configService,
+                   client: wc,
+                   logger: l,
+                   p: ps,
+                   configData: new ConfigurationDataBasicLogin())
+        {
+            Encoding = Encoding.UTF8;
+            Language = "ru-ru";
+            Type = "semi-private";
+        }
```

Comment:
you need to define a category
```suggestion
        }            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime");

```